omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -191,6 +193,69 @@
         </p>
     </div>
 
+
```

Comment:
Let's wrap this with a `<section>` at the top level. Right now it looks to me like it's bleeding off to the right (let me know that's not what you see on your screen).